uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/mocks/data/entitlement-data.js

```diff
@@ -0,0 +1,141 @@
+//contains data for the mock api calls for entitlement
+define(function() {
+  return {
+    all_data: {
+    "is_error": 0,
+    "version": 3,
+    "count": 6,
+    "values": [
+        {
+            "id": "1",
+            "period_id": "1",
+            "type_id": "1",
+            "contact_id": "202",
+            "overridden": "0"
+        },
+        {
+            "id": "2",
+            "period_id": "1",
+            "type_id": "2",
+            "contact_id": "202",
+            "overridden": "0"
+        },
+        {
+            "id": "3",
+            "period_id": "1",
+            "type_id": "3",
+            "contact_id": "202",
+            "overridden": "0"
+        },
+        {
+            "id": "4",
+            "period_id": "2",
+            "type_id": "1",
+            "contact_id": "202",
+            "overridden": "0"
+        },
+        {
+            "id": "5",
+            "period_id": "2",
+            "type_id": "2",
+            "contact_id": "202",
+            "overridden": "0"
+        },
+        {
+            "id": "6",
+            "period_id": "2",
+            "type_id": "3",
+            "contact_id": "202",
+            "overridden": "0"
+        }
+    ]
+    },
+    all_data_with_remainder: {
+    "is_error": 0,
```

Comment:
The indentation here is all over the place. Here there is no indentation, later on you have 4-spaces indentation